.github/workflows/ci.yml

```diff
@@ -51,15 +52,28 @@ jobs:
       if: steps.cache-ext.outputs.cache-hit != 'true'
       working-directory: ./ext
       run: bash libyuv.cmd
+    - name: Build googletest
+      # TODO: if: steps.cache-ext.outputs.cache-hit != 'true'
+      working-directory: ./ext
+      run: bash googletest.cmd
+      # Note: "apt install googletest" is sometimes insufficient for find_package(GTest) so build in ext/ instead.
 
     - name: Prepare libavif (cmake)
       run: |
         mkdir build && cd build
-        cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DAVIF_CODEC_AOM=ON -DAVIF_LOCAL_AOM=ON -DAVIF_CODEC_DAV1D=ON -DAVIF_LOCAL_DAV1D=ON -DAVIF_CODEC_RAV1E=ON -DAVIF_LOCAL_RAV1E=ON -DAVIF_CODEC_SVT=ON -DAVIF_LOCAL_SVT=ON -DAVIF_CODEC_LIBGAV1=ON -DAVIF_LOCAL_LIBGAV1=ON -DAVIF_LOCAL_LIBYUV=ON -DAVIF_BUILD_EXAMPLES=ON -DAVIF_BUILD_APPS=ON -DAVIF_BUILD_TESTS=ON
+        cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF \
+          -DAVIF_CODEC_AOM=ON -DAVIF_LOCAL_AOM=ON \
+          -DAVIF_CODEC_DAV1D=ON -DAVIF_LOCAL_DAV1D=ON \
+          -DAVIF_CODEC_RAV1E=ON -DAVIF_LOCAL_RAV1E=ON \
+          -DAVIF_CODEC_SVT=ON -DAVIF_LOCAL_SVT=ON \
+          -DAVIF_CODEC_LIBGAV1=ON -DAVIF_LOCAL_LIBGAV1=ON \
+          -DAVIF_LOCAL_LIBYUV=ON \
+          -DAVIF_BUILD_EXAMPLES=ON -DAVIF_BUILD_APPS=ON \
+          -DAVIF_BUILD_TESTS=1 -DAVIF_BUILD_GTESTS=1
```

Comment:
they have the same meaning, but I guess for consistency with the rest of the command line =ON might be more appropriate